src/scripts/config/index.ts

```diff
@@ -822,6 +822,72 @@ export const PF2ECONFIG = {
         Infinity: 4,
     },
 
+    terrainTypes: {
+        acidic: "PF2E.Terrain.Acidic",
+        aquatic: "PF2E.Terrain.Aquatic",
+        arctic: "PF2E.Terrain.Arctic",
+        blood: "PF2E.Terrain.Blood",
+        boat: "PF2E.Terrain.Boat",
+        cave: "PF2E.Terrain.Cave",
+        crowd: "PF2E.Terrain.Crowd",
+        debris: "PF2E.Terrain.Debris",
+        "decaying-plants": "PF2E.Terrain.DecayingPlants",
+        desert: "PF2E.Terrain.Desert",
+        dreamlands: "PF2E.Terrain.Dreamlands",
+        earth: "PF2E.Terrain.Earth",
+        "flower-field": "PF2E.Terrain.FlowerField",
+        foliage: "PF2E.Terrain.Foliage",
+        forest: "PF2E.Terrain.Forest",
+        fungus: "PF2E.Terrain.Fungus",
+        grass: "PF2E.Terrain.Grass",
+        grassland: "PF2E.Terrain.Grassland",
+        ice: "PF2E.Terrain.Ice",
+        jungle: "PF2E.Terrain.Jungle",
+        junk: "PF2E.Terrain.Junk",
+        lair: "PF2E.Terrain.Lair",
+        lava: "PF2E.Terrain.Lava",
+        lazurite: "PF2E.Terrain.Lazurite",
+        "ley-line": "PF2E.Terrain.LeyLine",
+        moist: "PF2E.Terrain.Moist",
+        mountain: "PF2E.Terrain.Mountain",
+        mud: "PF2E.Terrain.Mud",
+        natural: "PF2E.Terrain.Natural",
+        ocean: "PF2E.Terrain.Ocean",
+        outside: "PF2E.Terrain.Outside",
+        plains: "PF2E.Terrain.Plains",
+        plant: "PF2E.Terrain.Plant",
+        raining: "PF2E.Terrain.Raining",
+        rocky: "PF2E.Terrain.Rocky",
+        rubbish: "PF2E.Terrain.Rubbish",
+        rubble: "PF2E.Terrain.Rubble",
+        ruins: "PF2E.Terrain.Ruins",
+        rust: "PF2E.Terrain.Rust",
+        sand: "PF2E.Terrain.Sand",
+        scrapyard: "PF2E.Terrain.Scrapyard",
+        settlement: "PF2E.Terrain.Settlement",
+        sewer: "PF2E.Terrain.Sewer",
+        sky: "PF2E.Terrain.Sky",
+        smoke: "PF2E.Terrain.Smoke",
+        snow: "PF2E.Terrain.Snow",
+        stealth: "PF2E.Terrain.Stealth",
+        stone: "PF2E.Terrain.Stone",
+        swamp: "PF2E.Terrain.Swamp",
+        swardlands: "PF2E.Terrain.Swardlands",
+        "tar-pit": "PF2E.Terrain.TarPit",
+        trash: "PF2E.Terrain.Trash",
+        treasure: "PF2E.Terrain.Treasure",
+        tree: "PF2E.Terrain.Tree",
+        tundra: "PF2E.Terrain.Tundra",
+        underground: "PF2E.Terrain.Underground",
+        undergrowth: "PF2E.Terrain.Undergrowth",
+        underwater: "PF2E.Terrain.Underwater",
+        urban: "PF2E.Terrain.Urban",
+        vegetation: "PF2E.Terrain.Vegetation",
+        water: "PF2E.Terrain.Water",
+        "water:running": "PF2E.Terrain.RunningWater",
+        wilderness: "PF2E.Terrain.Wilderness",
+    },
```

Comment:
This is a mix of terrain types and environmental features

Reply:
I'm also noticing on a second glance that there's some standardization issues. What's the difference between urban and settlement?